prime-router/src/test/kotlin/fhirengine/translation/hl7/schema/ConfigSchemaReaderTests.kt

```diff
@@ -89,14 +89,15 @@ class ConfigSchemaReaderTests {
     @Test
     fun `test read schema tree from file`() {
         // This is a good schema
-        val schema = ConfigSchemaReader.readSchemaTreeRelative(
-            "ORU_R01",
-            "src/test/resources/fhirengine/translation/hl7/schema/schema-read-test-01"
+        val schema = ConfigSchemaReader.readSchemaTreeUri(
```

Comment:
This should still work... the existing code supports both relative and reading by absolute URI.

Reply:
why still keep the relative path tests since we are switching to schema reference with resource URI?

think some legacy tests either be refactored or replaced with URI based

Reply:
Mostly just to keep the scope of the work smaller and making rollback easier if we need to an update